core/src/test/java/com/linecorp/armeria/common/stream/StreamMessageTest.java

```diff
@@ -269,6 +276,30 @@ public void onComplete() {
         await().untilAsserted(() -> assertThat(buf.refCnt()).isZero());
     }
 
+    @TempDir
+    static Path tempDir;
```

Comment:
Can we remove this or just move inside of `test`?
(If this static variable is necessary, then we should locate it on top of this class same with other variables)